libraries/helpers.rb

```diff
@@ -195,6 +203,15 @@ def postgres_cfg_from_resource
       obj
     end
 
+    def active_directory_from_resource
+      spec = {}
+      spec['servers'] = new_resource.servers
```

Comment:
I think `servers` is probably too generic and may cause confusion when used outside of the property itself, how about `ad_servers`? You could internally still use servers within the resource itself as the ambiguity is only apparent outside of the direct resource code.

Reply:
I've changed it to ad_servers. I think it might be better as servers, however--we've been keeping a 1:1 correspondence between the Sensu Go documentation and property names for resources so far, with the only exception being `filter_action` since action is a reserved Chef word.

Reply:
Ya that makes sense, we could create the alias so that when people externally use it they can use `servers` or `ad_servers` while keeping the code with fewer ambiguities.